iOS/issue-tracker/issue-tracker/View/IssueTableViewCell.swift

```diff
@@ -0,0 +1,8 @@
+import UIKit
+
+class IssueTableViewCell: UITableViewCell {
+    static var nib: UINib {
+        return UINib(nibName: identifier, bundle: nil)
+    }
+    static let identifier = "IssueTableViewCell"
+}
```

Comment:
> 코드 형식

```suggestion
class IssueTableViewCell: UITableViewCell {

    static var nib: UINib {
        return UINib(nibName: identifier, bundle: nil)
    }
    static let identifier = "IssueTableViewCell"
    
}
```

=> 다른 파일들은 앞 뒤가 띄워져있어서 이렇게 제안해봅니다. pull request 보내기 전에 코드 형식이 잘 맞춰져있는지 확인해보세요 👍
다른 파일들도 코드 형식이 맞지 않은 파일이 있는지 확인하시고 피드백 반영 부탁드립니다.

Reply:
넵! 모두 확인하고 바꾸겠습니다 !